### DISCUSSION TITLE
Fallback crate redirect like npmjs.com and yarn.pm

### DISCUSSION THREAD

**Discussion by Fyko** [rust-lang/crates.io#7304] [Feature Requests]
Would be nice to have a default redirect for crates such as:
- https://crates.io/syn -> https://crates.io/crates/syn

similar to
- https://npmjs.com/lodash and 
- https://yarn.pm/lodash.

I made https://github.com/sycertech/crates.pm in the meantime but it would be lovely to have it natively.

**Comment by Turbo87** [comment]
what do we do if we want to introduce a new subpage of crates.io itself?

**Reply by Fyko** [reply]
Is this a question regarding SEO or user experience? If its the former, Google treats 301 and 302 the same. Latter, "oh well".
